src/Autosuggest.js

```diff
@@ -274,10 +280,19 @@ class Autosuggest extends Component {
       }
     };
     const onMouseEnter = (event, { sectionIndex, itemIndex }) => {
-      updateFocusedSuggestion(sectionIndex, itemIndex);
+      if (!focusedUsingKeyboard) {
+        updateFocusedSuggestion(sectionIndex, itemIndex);
+      }
     };
     const onMouseLeave = () => {
-      updateFocusedSuggestion(null, null);
+      if (!focusedUsingKeyboard) {
+        updateFocusedSuggestion(null, null);
+      }
+    };
+    const onMouseMove = () => {
+      if (focusedUsingKeyboard) {
+        updateFocusedUsingKeyboard(false);
+      }
```

Comment:
I wonder if running `updateFocusedUsingKeyboard(false)` in a `setTimeout` after `updateFocusedSuggestion(newFocusedSectionIndex, newFocusedItemIndex, value, true);` would work?
Just looking for ways to avoid adding `onMouseMove`.
What do you think?